ubidots/ubidots_out.html

```diff
@@ -1,102 +1,149 @@
+            </li>
+            <li>
+                <b>msg.payload</b> (optional) - The values to be published on the given Device. Each key is the label of the variable.
+                <p>Example of a simple value JSON message:<pre>{"value": 100}</pre>
+                <p>Example of a JSON message providing context data: <pre> {"variable": {"value": 200, "context": {"key1": "value1","key2": "value2"}}}</pre>
+                <p>Example of JSON message with multiple variables: <pre>{"variable_1": {"value": 100, "context": {"key1": "value1", "key2": "value2"}}, "variable_2": {"value": 200, "context": { "key1": "value1", "key2": "value2"}}} </pre>
 
-    <ul>
-        <li>
-            <b>msg.token</b> (required) - The token of your account.
-        </li>
-        <li>
-            <b>msg.device_label</b> (optional) - The name of the Device in which the packet will be published. If
-            no Device exists with this label, it will be automatically created.
-        </li>
-        <li>
-            <b>msg.payload</b> (optional) - The values to be published on the given Device. Each key is the label of the
-            variable.
-        </li>
-    </ul>
+            </li>
+        </ul>
     </p>
 
     <p>
-    Refer to the <a href="https://ubidots.com/docs/api/mqtt.html" target="_blank">Ubidots documentation</a> for more
-    information about the functionality of the MQTT broker.
+        Refer to the <a href="https://ubidots.com/docs/api/mqtt.html" target="_blank">Ubidots documentation</a> for more information about the functionality of the MQTT broker.
     </p>
 </script>
 
 <script type="text/javascript">
-    RED.nodes.registerType('ubidots_out', {
-        category: 'output',
-        color : "#5996F7",
-        defaults : {
-            name : {value: ""},
-            token : {value: ""},
-            label_device: {value: ""},
-            device_label: {value: ""},
-            tier: { value: 'business' }
-        },
-        inputs : 1,
-        outputs : 0,
-        icon : "ubidots.png",
-        label : function() {
-            return this.name || this.to || "ubidots out";
-        },
-        labelStyle : function() {
-            return this.name ? "node_label_italic" : "";
-        }
-    });
+  RED.nodes.registerType("ubidots_out", {
+    category: "output",
+    color: "#5996F7",
+    defaults: {
+      name: {
+        value: ""
+      },
+      token: {
+        value: "",
+        required: true
+      },
+      label_device: { value: "" },
+      device_label: {
+        value: ""
+      },
+      tier: {
+        value: "business"
+      },
+      tls_checkbox: { value: true }
+    },
+    inputs: 1,
+    outputs: 0,
+    icon: "ubidots.png",
+
+    label: function() {
+      return this.name || this.to || "Ubidots out";
+    },
+    labelStyle: function() {
+      return this.name ? "node_label_italic" : "";
+    },
+    oneditprepare: function() {},
```

Comment:
Just to make sure, this is required? If you remove this property which has a noop function it still works OK?

If it does, please remove it